blueprints/app/files/tests/helpers/index.ts

```diff
@@ -2,7 +2,7 @@ import {
   setupApplicationTest as upstreamSetupApplicationTest,
   setupRenderingTest as upstreamSetupRenderingTest,
   setupTest as upstreamSetupTest,
-  SetupTestOptions,
+  type SetupTestOptions,
```

Comment:
`@tsconfig/ember` uses `verbatimModuleSyntax` which requires type imports to be declared